README.md

```diff
@@ -378,20 +363,40 @@ actions: {
 }
 ```
 
+=======
```

Comment:
Would be nice to remove this and following conflict artefacts 😉 Otherwise [`README` looks weird](https://github.com/bendemboski/ember-cli-bootstrap-datepicker/blob/6fe2e2c20ed2b6e3531df6de7f72a79b0b1e3228/README.md#contributing).

Reply:
Actually I was thinking it would be helpful to users to see fragments of the two different versions in the conflict blocks.

Reply:
Kidding ;)

That's embarrassing! Just pushed a commit resolving.

Reply:
Thanks! 😄